functions.txt

```diff
@@ -611,7 +611,7 @@ function ord ($c ::: string) ::: int;
 function strcasecmp ($str1 ::: string, $str2 ::: string) ::: int;
 function strcmp ($str1 ::: string, $str2 ::: string) ::: int;
 function stripslashes ($str ::: string) ::: string;
-function strip_tags ($str ::: string, $allow ::: string = "") ::: string;
+function strip_tags ($str ::: string, $allow ::: string|string[] = "") ::: string;
```

Comment:
Please, consider the following example

```php
<?php

function demo($s, $t) {
  strip_tags($s, $t);
}

demo("x", ["y"]);
demo("x", "z");
```

If you compile it, it will produce C++ compilation error:
```
error: no matching function for call to ‘f$strip_tags(const string&, const mixed&)’
```

Reply:
I'll probably add a mixed argument overloading there and then I'll remove WIP prefix.

Reply:
Added `mixed` and `array<Unknown>` overloadings.
`array<Unknown>` is needed for empty arrays.